migrations/2_deploy_supply_chain.js

```diff
@@ -0,0 +1,7 @@
+/* global artifacts */
```

Comment:
You can set globals in eslint config.

Reply:
You're right, for migrations and tests it's probably better to go with config globals instead of declarations on top of a file.